week-2/presentation.html

```diff
@@ -37,7 +37,6 @@
             <textarea data-template>
             ## Coaches
             - Dimitri Roose
-            - Randi Vertongen
```

Comment:
this could also be generalized to "coaches introduce themselves" with no specific names.  then we don't need to update this slide for future classes/locations

Reply:
Approached it how @colevandersWands suggested.